### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by trasherdk**
Initial `npm ci`
- Add `deprecated.md`

caniuse-lite - make browserlist shut up.

Bump babel stuff to latest
- `babel-eslint` to `@babel/eslint-parser`

Bump `eslint` to `8.4.1`
- remove outdated `eslint-plugin-ante`

Bump `mocha 8.2.1 → 9.1.3`

Fix the last warning `path-parse`
- remove `deprecated.md`

Remove `deprecated.md` for real this time.

Somehow `fsevents` got bumped in `yarn.lock`

### CODE REVIEW

**Comment by trasherdk** [comment]
I would run `standard` on this thing, to conform with `npm`, but that will touch a lot of files.

File: .eslintrc.js [hunk 1]
```diff
@@ -17,7 +15,6 @@ module.exports = {
     sourceType: 'module'
   },
   plugins: [
-    'eslint-plugin-ante'
```
Comment (canterberry) [inline .eslintrc.js]: I pushed up a new version of this plugin, so it should no longer complain about an ESLint incompatibility.

**Review by canterberry — CHANGES_REQUESTED** [review]
Thanks for this housekeeping PR. Been a while since I ran a round of routine upgrades.

Please add eslint-plugin-ante back in, now that it's been adjusted to accommodate the current ESLint version and it's what enforces all the lint stuff including code style.

Also, please adjust **package.json** version strings to remove the `^`.

Aside from those two details, looks good to me.

File: .eslintrc.js (hunk 1 above)
Comment (trasherdk) [reply .eslintrc.js]: The 'eslint-plugin-ante' thingy is back.

**Comment by trasherdk** [comment]
There has been *a few* updates since this PR was compiled.

```
 @babel/cli              7.16.0  →   7.17.0     
 @babel/core             7.16.5  →   7.17.2     
 @babel/eslint-parser    7.16.5  →   7.17.0     
 @babel/preset-env       7.16.5  →  7.16.11     
 @babel/register         7.16.5  →   7.17.0     
 @types/node           14.14.22  →  17.0.17     
 eslint                   8.4.1  →    8.9.0     
 mocha                    9.1.3  →    9.2.0     
```
I could push on top of this.

**Pull request closed without merging**